api/get_user_states.go

```diff
@@ -54,5 +53,24 @@ func (a *API) unsafeGetUserStates(ctx context.Context, publicKey *models.PublicK
 		userStates = append(userStates, dto.MakeUserStateWithID(stateID, pendingState))
 	}
 
+	pendingUserStates, err := a.storage.GetPendingUserStates(publicKey)
```

Comment:
ok, this only returns states not yet covered by `GetPendingUserState` (singular)